frontend/src/components/Forms/ShoutoutsPage/ShoutoutCard.tsx

```diff
@@ -1,18 +1,37 @@
-import { Card } from 'semantic-ui-react';
+import React, { useState, useEffect } from 'react';
+import { Card, Image, Loader } from 'semantic-ui-react';
 import ShoutoutDeleteModal from '../../Modals/ShoutoutDeleteModal';
 import styles from './ShoutoutCard.module.css';
+import ImagesAPI from '../../../API/ImagesAPI';
 
 const ShoutoutCard = (props: {
   shoutout: Shoutout;
   setGivenShoutouts: React.Dispatch<React.SetStateAction<Shoutout[]>>;
 }): JSX.Element => {
   const { shoutout, setGivenShoutouts } = props;
 
+  const [image, setImage] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
   const fromString = shoutout.isAnon
     ? 'From: Anonymous'
     : `From: ${shoutout.giver?.firstName} ${shoutout.giver?.lastName}`;
```

Comment:
I'm wondering if there could be a case where either the firstname and the lastName are null/undefined? Would be helpful to take these cases and provide a fallback for both in that case!